helm/defectdojo/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 16.7.0
+  version: 16.7.26
 - name: redis
```

Comment:
We're currently evaluating migrating to ValKey so let's not bump this dependency yet.